src/metal.cs

```diff
@@ -1741,6 +2001,69 @@ partial interface MTLRenderCommandEncoder : MTLCommandEncoder {
 #endif
 		[Export ("drawIndexedPatches:patchIndexBuffer:patchIndexBufferOffset:controlPointIndexBuffer:controlPointIndexBufferOffset:indirectBuffer:indirectBufferOffset:")]
 		void DrawIndexedPatches (nuint numberOfPatchControlPoints, [NullAllowed] IMTLBuffer patchIndexBuffer, nuint patchIndexBufferOffset, IMTLBuffer controlPointIndexBuffer, nuint controlPointIndexBufferOffset, IMTLBuffer indirectBuffer, nuint indirectBufferOffset);
+
+		[Mac (10,13), NoiOS, NoTV, NoWatch]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("setViewports:count:")]
+		void SetViewports (IntPtr viewports, nuint count);
```

Comment:
OK, sounds like the developer will never end up overriding/implementing this protocol, which makes it easier to swallow the IntPtr API.